### DISCUSSION TITLE
Is there a way to add vertical line separators between columns in FluentUI data grid?

### DISCUSSION THREAD

**Discussion by cilerler** [microsoft/fluentui-blazor#2929] [Q&A] [answered]
We recently switched to FluentUI for our data grid and miss having vertical line separators between columns for better readability. Is there a way to add these in the FluentUI data grid?

## Expected
![image](https://github.com/user-attachments/assets/3851619f-ed59-4b6d-8d31-0ea994f5d4df)

## Actual
![image](https://github.com/user-attachments/assets/d740b896-5269-4659-9ec0-3ec1acb2e0ed)

**Comment by dvoituron** [comment, marked as answer]
Not really, but you can apply a custom style to the Cells:
```css
fluent-data-grid-cell {
    border-left: 1px solid silver;
    border-radius: unset;
```

**Reply by cilerler** [reply]
Thank you very much
